src/language-js/parse/babel.js

```diff
@@ -111,11 +111,20 @@ function createParse(parseMethod, ...optionsCombinations) {
     }
 
     if (text.includes("|>")) {
-      combinations = pipelineOperatorPlugins.flatMap((pipelineOperatorPlugin) =>
+      const conflictsPlugins = pipelineOperatorPlugins;
```

Comment:
Need copy array here.

Reply:
This is not fixed, you are still manipulating to the `pipelineOperatorPlugins` array.

Reply:
Ah sorry I misunderstood. Okay, I'll fix it.